modules/administration/pages/tshoot-clienthang.adoc

```diff
@@ -0,0 +1,25 @@
+[[troubleshooting-client-hang]]
+= Troubleshooting Client Hanging on Reboot
+
+////
+PUT THIS COMMENT AT THE TOP OF TROUBLESHOOTING SECTIONS
+
+Troubleshooting format:
+
+One sentence each:
+Cause: What created the problem?
+Consequence: What does the user see when this happens?
+Fix: What can the user do to fix this problem?
+Result: What happens after the user has completed the fix?
+
+If more detailed instructions are required, put them in a "Resolving" procedure:
+.Procedure: Resolving Widget Wobbles
+. First step
+. Another step
+. Last step
+////
+
+Sometimes when attempting to reboot a Salt minion after applying a formula, the terminal will hang at the boot screen.
```

Comment:
I would recommend only a few minor changes - dropping "after applying a formula" (Saltboot formula isn't applied at all because of timeout value is set too low, so this is the cause of hanging at the boot screen.) And maybe we should be more specific about the type of client, this can happen only in case of retail terminal. ("Salt minion retail terminal"?)

Reply:
If this only applies to Retail, then I'm going to move it to the Retail book.